src/meta/MetaHttpDownloadHandler.cpp

```diff
@@ -165,34 +174,50 @@ bool MetaHttpDownloadHandler::dispatchSSTFiles(const std::string& hdfsHost,
         return false;
     }
 
-    std::atomic<int> completed(0);
-    std::vector<std::thread> threads;
+    std::vector<folly::SemiFuture<bool>> futures;
+    static nebula::thread::GenericThreadPool pool;
+    std::call_once(downloadPoolStartFlag, []() {
```

Comment:
Maybe you can define downloadPoolStartFlag with `static`, no need to define as global, just a suggestion.

Reply:
👍  fixed